pkgs/development/tools/parsing/tree-sitter/default.nix

```diff
@@ -166,6 +211,10 @@ rustPlatform.buildRustPackage {
       * Fast enough to parse on every keystroke in a text editor
       * Robust enough to provide useful results even in the presence of syntax errors
       * Dependency-free so that the runtime library (which is written in pure C) can be embedded in any application
+
+      In order to use the command-line tool, import pkgs.tree-sitter.withAllGrammarSources, or pick individual languages with
```

Comment:
IMO "import" is not the best choice of word, because it is a nix keyword and you can't `import` this package. Maybe "install" instead, or even "use".